### PR TITLE
Remove osmodel.mak in favor of ../dmd/osmodel.mak

### PR DESCRIPTION
**Pull request by wilzbach**
A small step at cleaning up the Makefiles.

Similarly to https://github.com/dlang/tools/pull/227

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -31,8 +31,11 @@
 QUIET:=
 
 DEBUGGER=gdb
+GIT_HOME=https://github.com/dlang
+DMD_DIR=../dmd
 
-include osmodel.mak
+$(shell [ ! -d $(DMD_DIR) ] && git clone --depth=1 $(GIT_HOME)/dmd $(DMD_DIR))
```
Comment (wilzbach) [inline posix.mak]: @CyberShadow this was needed to pass on DAutoTest. I assumed that you clone all repos on start?
Anyhow I doubt that this hurts anyone and at best it helps a few newcomers.

Comment (CyberShadow) [inline posix.mak]: > I assumed that you clone all repos on start?

They're always there, just cleaned/reset between runs. Not sure why you'd get an error there.

**Comment by wilzbach** [comment]
Btw it seems as if we got four new "random" coverage lines:

image

**Comment by CyberShadow** [comment]
At least it's in paralellism, I won't feel guilty for masking those out.

File: posix.mak (hunk 1 above)
Comment (andralex) [inline posix.mak]: Let's see why the simpler solution doesn't work, this is an unpleasant complication.

**Comment by wilzbach** [comment]
> They're always there, just cleaned/reset between runs. Not sure why you'd get an error there.
> Let's see why the simpler solution doesn't work, this is an unpleasant complication.

Alrighty - seem to have been a transient error. Removed the line & everything* is green  :)

* except for the broken Project-Tester of course.

**Comment by CyberShadow** [comment]
> Not sure why you'd get an error there.

On second thought, I know why.

https://github.com/CyberShadow/ae/commit/0cc44962db38dbadd194850e7d86c1928a8d72fc

**Comment by CyberShadow** [comment]
> which as far as I can see never existed

Oops, that's not true. Still looking into it

**Comment by wilzbach** [comment]
>  I think because of the above (now-fixed) bug we slip a build bug in, and now website auto-deploy is failing.

Sorry.

> Oops, that's not true. Still looking into it

I think I can submit a PR. Continue on your bug hunt ;-)

**Comment by wilzbach** [comment]
> I think I can submit a PR

https://github.com/dlang/dlang.org/pull/1791

**Pull request merged**